readthedocs/templates/doc_builder/conf.py.tmpl

```diff
@@ -32,12 +36,22 @@ if globals().get('source_suffix', False):
 else:
     SUFFIX = '.rst'
 
+# Support markdown
+def setup(app):
+    app.add_source_parser('.md', CommonMarkParser)
+    app.add_config_value('recommonmark_config', {
+            'enable_eval_rst': True
+        }, True)
+    app.add_transform(AutoStructify)
+
+exclude_patterns = ['readme.md', 'README.md']
```

Comment:
Potremmo escludere anche 'LICENSE.md' e 'AUTHORS.md' (con le varianti lowcase), giusto per stare tranquilli

Reply:
Si giusto! Provvedo subito.

Reply:
In ogni caso questo serve solo a prevenire dei warning rispetto al fatto che i file con estensione md non sono stati inclusi da index.